### PR TITLE
feat: support using current directory name as env name

### PR DESCRIPTION
**Pull request by fatelei**
#920 

image

### CODE REVIEW

**Review by gaocegege — COMMENTED** [review]
Thanks for your contribution! :tada: :+1:

/assign @VoVAllen

File: pkg/app/env_describe.go [hunk 1]
```diff
@@ -27,16 +28,24 @@ import (
 	"github.com/tensorchord/envd/pkg/types"
 )
 
+func getCurrentDir() string {
```
Comment (VoVAllen) [inline pkg/app/env_describe.go:31]: Rename to getCurrentDirOrPanic

**Review by VoVAllen — COMMENTED** [review]
Thanks for your contribution. One minor comment here

**Comment by gaocegege** [comment]
@all-contributors Please add @fatelei for the code contribution.

**Pull request merged**